ethportal-peertest/src/jsonrpc.rs

```diff
@@ -113,7 +157,7 @@ pub async fn test_jsonrpc_endpoints_over_ipc(target_ipc_path: String) {
                 ),
             };
             match get_response_result(response_obj) {
-                Ok(result) => validate_endpoint_response(endpoint.method, &result),
+                Ok(result) => validate_endpoint_response(endpoint.method.as_str(), &result),
```

Comment:
I don't have a strong sense of when's the right time to do use `as_str()` vs `&`, but I've tended to prefer `&` when possible.